install_julia.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+apt-get -qq update
+DEBIAN_FRONTEND=noninteractive apt-get -qqy install ${CI_APT_INSTALL:-}
+
+if [[ "$1" == "source" ]]; then
+  DEBIAN_FRONTEND=noninteractive apt-get -qqy install build-essential cmake curl gfortran git libatomic1 m4 perl pkg-config python rsync > /dev/null
```

Comment:
why the pipe to /dev/null here but not in other cases?

Reply:
Oversight, I'll need to add it